DIGNDB.APP.SmitteStop.Jobs/Services/KeyNumbers/FetchKeyNumbersService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using DIGNDB.APP.SmitteStop.Jobs.Jobs.Interfaces;
+using DIGNDB.APP.SmitteStop.Jobs.Services;
+
+namespace DIGNDB.APP.SmitteStop.Jobs.Jobs.KeyNumbers
+{
+    public class FetchKeyNumbersService : IFetchKeyNumbersService
+    {
+
+        private readonly IKeyNumbersWebService keyNumbersWebService;
+
+        private const string BASE_URL = "https://raw.githubusercontent.com/folkehelseinstituttet/surveillance_data/master/covid19/";
+        private const string TESTED_URL = "data_covid19_lab_by_time_";
+        private const string HOSPITAL_ADMISSIONS_URL = "data_covid19_hospital_by_time_";
+        private const string VACCINATION_URL = "data_covid19_sysvak_by_time_location_";
+
+        public FetchKeyNumbersService(IKeyNumbersWebService webService)
+        {
+            keyNumbersWebService = webService;
+        }
+
+        public async Task<Stream> FetchTestNumbersFromDate(DateTime date)
+        {
+            return await keyNumbersWebService.GetAsync($"{BASE_URL}{TESTED_URL}{date:yyyy-MM-dd}.csv").Result.Content.ReadAsStreamAsync();
```

Comment:
Perhapns it would be a good idea to separate logic here - create generic method get response and read it as stream and isolate creation of file name from that method. However it is not a big thing here.